### ISSUE TITLE
Script installation failed on Arm64 server.

### ISSUE THREAD

**Issue by SFARPak** [Har-Kuun/OneClickCDN#18]
![image](https://user-images.githubusercontent.com/89567338/165526603-a7b24e1c-cb7c-41e2-b134-cbcc953ad08a.png)

**Comment by bitUpsurge** [comment]
I don't think it supports Arm Architecture for now, please contact developer for further information and help.
